### PR TITLE
chore: Clean up trailing spaces

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Details

Tool-driven change to remove trailing spaces that are scattered throughout the code

##### Motivation

I've been picking these off opportunistically with other changes, and decided to get them all in a single pass.

#### Pull request checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/main/docs/Scenarios.md) completed?
- [n/a] Does this address an existing issue? If yes, Issue# - 
- [n/a] Includes UI changes?
  - [n/a] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [n/a] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: src/AccessibilityInsights.Extensions.AzureDevOps/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("AccessibilityInsights.Extensions.AzureDevOps")]
-[assembly: AssemblyCopyright("Copyright �  2017")]
+[assembly: AssemblyCopyright("Copyright �  2017")]
```
Comment (jalkire) [inline src/AccessibilityInsights.Extensions.AzureDevOps/Properties/AssemblyInfo.cs:16]: This new copyright character isn't rendering in the diff for me, is this an intended change?

File: src/AccessibilityInsights.Extensions.Telemetry/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("AccessibilityInsights.Extensions.Telemetry")]
-[assembly: AssemblyCopyright("Copyright �  2017")]
+[assembly: AssemblyCopyright("Copyright �  2017")]
```
Comment (jalkire) [inline src/AccessibilityInsights.Extensions.Telemetry/Properties/AssemblyInfo.cs:15]: Same copyright character question

Comment (DaveTryon) [reply src/AccessibilityInsights.Extensions.Telemetry/Properties/AssemblyInfo.cs]: Nice catch! fixed

File: src/AccessibilityInsights.Extensions.AzureDevOps/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (DaveTryon) [reply src/AccessibilityInsights.Extensions.AzureDevOps/Properties/AssemblyInfo.cs]: Nice catch! fixed

**Review by jalkire — APPROVED** [review]

**Pull request merged**